File: php/testcode/benchmark_test_00202.php
<?php
require_once __DIR__ . '/shared.php';

function benchmarkTest00202(BenchmarkRequest $req): BenchmarkResponse {
    $reader = new XMLReader();
    $reader->open('/var/app/uploads/' . basename($req->param('file')));
    $reader->setParserProperty(XMLReader::SUBST_ENTITIES, false);
    return BenchmarkResponse::ok('configured');
}

File: rust/testcode/benchmark_test_00202.rs
pub fn handle(req: &super::shared::BenchmarkRequest) -> super::shared::BenchmarkResponse {
    let token = req.param("token");

    let cookie = format!("session={}; Path=/; SameSite=None", token);

    super::shared::BenchmarkResponse::ok(&format!("Set-Cookie: {}", cookie))
}
